projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -397,6 +385,59 @@ public function prepare_item_for_response( $item, $request ) {
 		return rest_ensure_response( $item );
 	}
 
+	/**
+	 * Checks that the "plugins" parameter is not empty.
+	 *
+	 * @param array           $plugins List of plugins to update.
+	 * @param WP_REST_Request $request Request object.
+	 * @param string          $param   The parameter name.
+	 * @return bool|WP_Error
+	 */
+	public function validate_plugins_param( $plugins, $request, $param ) {
+		$result = rest_validate_request_arg( $plugins, $request, $param );
+		if ( is_wp_error( $result ) ) {
+			return $result;
+		}
+
+		$installed_plugins = array_filter( $plugins, array( $this, 'is_plugin_installed' ) );
+		if ( empty( $installed_plugins ) ) {
+			add_filter( 'rest_request_after_callbacks', array( $this, 'transform_error_response' ) );
+
+			return new \WP_Error(
+				'rest_forbidden',
```

Comment:
Should we use something else instead since we're moving this into a validate callback check and it returns 400 not 403 anymore?